testable-simd-models/src/helpers.rs

```diff
@@ -2,6 +2,15 @@
 pub mod test {
     use crate::abstractions::{bit::Bit, bitvec::BitVec, funarr::FunArray};
     use rand::prelude::*;
+    use std::sync::Mutex;
```

Comment:
Seems fine to me: Cargo runs each #[test] in it's own process I believe.
Howover, you don't need lazy_static, you can use LazyLock from std.

Reply:
Ah, and shall we have a optional environment variable to set the seed explicitely?